scripts/get_grafana_dashboards.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+# https://istio.io/latest/docs/ops/integrations/grafana/#option-1-quick-start
+for DASHBOARD in 7639 11829 7636 7630 7642 7645; do
```

Comment:
I think this isn't getting called anywhere? does it need to be added to the run.sh?

Reply:
This script is executed during the docker build. I added it there so that the images are downloaded during build time to prevent the installation from failing because of grafana.com being down or not reachable in an airgapped environment.

Reply:
I do see that now - I read too quickly and thought they were both mkdir 🤦